deepchem/models/graph_models.py

```diff
@@ -715,7 +715,7 @@ class _GraphConvKerasModel(tf.keras.Model):
   def __init__(self,
                n_tasks,
                graph_conv_layers,
-               dense_layer_size=128,
+               dense_layers, # changed dense_layer_size to dense_layers as a list instead
                dropout=0.0,
```

Comment:
Could you add options for weight/bias initialization, batch normalization, and weight decay for fully connected layers as in `WeaveModel`? (https://github.com/deepchem/deepchem/blob/master/deepchem/models/graph_models.py#L89)

Reply:
Ah we already have batch-norms and dropouts so we're good on that front